examples/09_mpi_units.py

```diff
@@ -34,8 +34,12 @@
 
     # use the resource specified as argument, fall back to localhost
     if   len(sys.argv)  > 2: report.exit('Usage:\t%s [resource]\n\n' % sys.argv[0])
-    elif len(sys.argv) == 2: resource = sys.argv[1]
-    else                   : resource = 'local.localhost'
+    elif len(sys.argv) == 2: 
+        resource = sys.argv[1]
+        cu_arg = ['09_mpi_units.sh']
+    else: 
+        resource = 'local.localhost'
+        cu_arg = ['%s/09_mpi_units.sh' % PWD]
```

Comment:
Since staging is also performed on localhost, shouldn't `cu_arg = ['09_mpi_units.sh']` work for both cases?

Reply:
Also, `sys.argv[1]` can also be the string `local.localhost`.